querybook/webapp/ui/Comment/Reactions.tsx

```diff
@@ -39,6 +49,7 @@ export const Reactions: React.FunctionComponent<IProps> = ({
         const existingReaction = reactionsByEmoji[emoji].find(
             (reaction) => reaction.created_by === uid
         );
+        setIsLoadingEmoji(emoji);
```

Comment:
make `setIsLoadingEmoji(emoji);` and `setIsLoadingEmoji(null)` in the same function. you can do something like
```
setIsLoadingEmoji(emoji);
try {
        if (existingReaction) {
            await deleteEmoji(existingReaction.id);
        } else {
            await addEmoji(emoji);
        }
} finally {
       setIsLoadingEmoji(null)
}
```
or do it in addEmoji and deleteEmoji respectively